resources/js/components/StartExperimentBtn.vue

```diff
@@ -20,7 +20,9 @@
 			    </div>
 			  </div>
 		</div>
-		
+		<span v-if="timeexpires==true" class="start" style="position: absolute;top:45%; left: 45%;  cursor: pointer;background: #a23;color: #fff;padding: 15px 20px;border-radius: 8px;">
```

Comment:
@saidabdul80 Why are we writing inline CSS instead of write of a class for it? Remember Vue lets you write scoped style if you think you will only be using the class in this file

Reply:
resolved in next PR sir please pardon and merge